package/base-files/files/lib/upgrade/common.sh

```
@@ -91,7 +91,7 @@ get_image() { # <source> [ <command> ]
 		esac
 	fi
 
-	cat "$from" 2>/dev/null | $cmd
+	$cmd "$from" 2>/dev/null
```

This will suppress errors for $cmd, while the first one only suppressed errors from cat.

---

I dropped the redirection.

---

Since this is upgrade code, and this is still a behavior change, this should be reviewed and tested carefully.